estate_account/models/estate_property.py

```diff
@@ -0,0 +1,26 @@
+from odoo import Command, models
+
+
+class EstateProperty(models.Model):
+    _inherit = 'estate.property'
+
+    def action_sold(self):
+        """
+        Create an invoice for the property sale
+        """
+        result = super().action_sold()
+        if not result:
+            return result
+
+        self.env['account.move'].create(
+            {
```

Comment:
```suggestion
        self.env['account.move'].create({
```
Depending on the team.
Up to you.